test/dune

```diff
@@ -0,0 +1,11 @@
+(test
+ (name test)
+ (modules test)
+ (flags :standard -thread)
+ (libraries alcotest bigstring bigstring-unix threads)
+ (action
+  (setenv MALLOC_CHECK_ 3
```

Comment:
interesting, is that specific to a toolchain? a comment would be helpful .

Reply:
I will add a comment. This is enabled by default on OpenBSD and it helped me to discover some nasty bugs early. See http://man7.org/linux/man-pages/man3/mallopt.3.html#DESCRIPTION